aptos-move/framework/aptos-framework/sources/object_code_deployment.move

```diff
@@ -0,0 +1,73 @@
+module aptos_framework::object_code_deployment {
+    use std::bcs;
+    use std::features;
+    use std::signer;
+    use std::string;
+    use std::vector;
+    use aptos_framework::account;
+    use aptos_framework::code;
+    use aptos_framework::code::PackageRegistry;
+    use aptos_framework::object;
+    use aptos_framework::object::{ExtendRef, Object};
+
+    /// Object code deployment not supported.
+    const EOBJECT_CODE_DEPLOYMENT_NOT_SUPPORTED: u64 = 0x1;
```

Comment:
I see in other modules (I checked object.move) that we just use normal numbers here, e.g.
```
const ENOT_OBJECT_OWNER: u64 = 4;
```

Can we do that here too?

Reply:
Was following the convention in [`code.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/code.move#L70) which uses this pattern I used. If there is a strong reason to use numbers then will change.